odoo/fields.py

```diff
@@ -3184,6 +3184,10 @@ class Properties(Field):
         'datetime', 'many2one', 'many2many', 'selection', 'tags',
     )
 
+    @property
+    def _description_sortable(self):
+        return False
+
```

Comment:
can't you just create an attribute `_description_sortable = False` on the class itself ?